22.3 fb clean up onprc ehr.xml

jonesgaohsu opened the pull request:
#### Rationale
Clean up of onprc_ehr.xml data per instructions from labkey to allow deletion of folders and workbooks
#### Related Pull Requests
Closer review of the recommendations from Josh revealed I had not compelted all the needed actions.  Needed to remove the onprc_ehr.XX.sql that created the files.  
#### Changes
* <!-- list of descriptions of changes that are worth noting (replace this comment) -->

jonesgaohsu committed:
Chaned the source of roomAssignbment for Finance Display

jonesgaohsu committed:
Chaned the source of roomAssignbment for Finance Display

jonesgaohsu committed:
Merge remote-tracking branch 'origin/release22.3-SNAPSHOT' into release22.3-SNAPSHOT

jonesgaohsu committed:
Cleaning up the missing xml files related per ticket https://www.labkey.org/ONPRC/Support%20Tickets/issues-details.view?issueId=46252

jonesgaohsu committed:
Cleaning up the missing xml files related per ticket https://www.labkey.org/ONPRC/Support%20Tickets/issues-details.view?issueId=46252

jonesgaohsu committed:
Cleaning up the missing xml files related per ticket https://www.labkey.org/ONPRC/Support%20Tickets/issues-details.view?issueId=46252

labkey-jeckels commented on onprc_ehr/resources/queries/ehr_lookups/roomsAssignment.sql:
```diff
@@ -34,7 +35,7 @@ select
   count(distinct a.id) as totalAnimals
 
-from study.assignment a
+from finance_Study.assignment a
```
This change seemed unrelated and accidentally included. Please remove.

labkey-jeckels commented on onprc_ehr/resources/schemas/onprc_ehr.xml:
```diff
@@ -763,72 +652,8 @@
             <column columnName="container"/>
         </columns>
     </table>
-    <table tableName="eIACUC_PRIME_VIEW_ANIMAL_GROUPS" tableDbType="TABLE">
```
This table, and the other eIACUC* tables, should be retained. The problematic ones are the ones that start with PRIME_VIEW

jonesgaohsu replied:
So should I change in intellij and then resubmit?

labkey-jeckels replied:
@jonesgaohsu yes please. From a first glance, your other followup commits here look correct.

jonesgaohsu replied:
Thanks for the support on this

jonesgaohsu replied:
I pulled down a fresh copy and made the needed corrections and will verify in test instance once the build is complete.  I believe the needed modifications and removals were made

labkey-jeckels commented on onprc_ehr/resources/schemas/onprc_ehr.xml:
```diff
@@ -763,72 +652,8 @@
-            <column columnName="Death_As_Endpoint_Justification"/>
-        </columns>
-    </table>
-    <table tableName="eIACUC_PRIME_VIEW_SURGICAL_PROCS" tableDbType="TABLE">
```
Should not be removed.

labkey-jeckels reviewed (changes requested):
We can't completely delete onprc_ehr-18.10-20.101.sql. We should keep the file but delete the statements that are creating the problematic project, protocol, and PRIME_VIEW tables.

jonesgaohsu committed:
Update roomsAssignment.sql

This should not have been included

jonesgaohsu committed:
Update per instructions from Labkey to leave in place but removed the create table satements that were causing an issue

jonesgaohsu committed:
Merge remote-tracking branch 'origin/22.3_fb_CleanUpOnprc_ehr.xml' into 22.3_fb_CleanUpOnprc_ehr.xml

jonesgaohsu committed:
REmoval of non eIACUC definitions and the definitions for protoco and project in onprc_ehr

labkey-jeckels commented on onprc_ehr/resources/schemas/dbscripts/sqlserver/onprc_ehr-18.10-20.101.sql:
```diff
@@ -114,115 +114,6 @@ GO
- *
- */
 
 CREATE TABLE [onprc_ehr].[PRIME_VIEW_PROTOCOL_Processing](
```
This one should get deleted too

jonesgaohsu replied:
OK will remove and reload

jonesgaohsu replied:
Copncentrating on clean up of onprc_ehr-18..10-20.101 and removing all reference to eiACUC processsing, this was 3 year old code will be updating the build in the next 30 minutes

jonesgaohsu replied:
Push completed My review shows all references to eIACUC processing has been removed

jonesgaohsu committed:
Update onprc_ehr-18.10-20.101.sql

I have cleared the code here, the stored procedure and associated table creations are not longer used

jonesgaohsu committed:
Cleared all references to eiACUC from here as those were old code no longer valie

jonesgaohsu committed:
Cleared all references to eiACUC from here as those were old code no longer valie

jonesgaohsu committed:
Cleared all references to eiACUC from here as those were old code no longer valie

labkey-jeckels commented on onprc_ehr/resources/web/onprc_ehr/panel/AssignmentFilterType.js:
```diff
@@ -65,7 +65,7 @@ Ext4.define('ONPRC_EHR.panel.AssignmentFilterType', {
                     type: 'labkey-store',
                     schemaName: 'onprc_ehr',
-                    sql: 'SELECT distinct lastname FROM onprc_ehr.investigators WHERE lastname is not null',
+                    sql: 'SELECT distinct lastname FROM onprc_ehr.investigators WHERE enddate is null',
```
This change seems unrelated. Is it something you want to include here?

jonesgaohsu replied:
NO let me remove it from the build

jonesgaohsu replied:
File restored to original state and submitted

labkey-jeckels replied:
@jonesgaohsu you added a comment but didn't actually back out the change to the SQL.

jonesgaohsu committed:
Removed changed to AssignmentFilterType.js

jonesgaohsu committed:
Update AssignmentFilterType.js

made change to line 68 returning the original code

jonesgaohsu commented:
Made the change on line 68 to return the code to original code
